### DISCUSSION TITLE
How to load a 3D virtual tour on ReactJS?

### DISCUSSION THREAD

**Discussion by OctavioKh** [jeffreylanters/react-unity-webgl#210] [Help Wanted!]
Hello! i've been trying to set up my unity game on a React application, and i've tried the following so far

```jsx
import React, {Component} from 'react';
import Game from './game.jsx';
import {Container} from 'react-bootstrap';

class GamePage extends Component {
  render() {
    return ( 
      <div>
        <Game/>
        <Container>
          //text and stuff inside the container
        </Container>
      </div>
    );
  }
}

export default GamePage;
```

Where game.jsx is : 

```jsx
import React from "react";
import Unity, { UnityContext } from "react-unity-webgl";

const unityContext = new UnityContext({
  loaderUrl: "../Build/unity341.loader.js",
  dataUrl: "../Build/unity341.data",
  frameworkUrl: "../Build/unity.framework.js",
  codeUrl: "../Build/unity341.wasm",
});

function Game() {
  return <Unity unityContext={unityContext} />;
}

export default Game;
```

I've also tried loading the scripts from the index.html, and trying to dangerously insert HTML  with the scripts from the build, but i'm unable to embed it on my  React Project  any help would be extremely appreciated..!

Thanks  

DOGEwiki

**Comment by jeffreylanters** [comment]
Hi, sorry for my late response. I've haven't been able to check or here recently.

Thanks for opening an issue. In the case you haven't solved your problem yet, what exactly is happening within your browser? Are you seeing any canvas at all, or do any errors pop-up withint the browsers inspector console? Also good and creative thinking, but loading the scripts via dangerously set innerHTML will not work since the module is waiting for the scripts to load successfully from within the module before mounting the Unity Player.
